src/python_routing_v02/compute_nhd_routing_SingleSeg_v02.py

```diff
@@ -845,7 +850,7 @@ def compute_nhd_routing_v02(
                     qts_subdivisions,
                     reaches_list_with_type,
                     independent_networks[tw],
-                    param_df_sub.index.values,
+                    param_df_sub.index.values.astype("int64"),
```

Comment:
Track down the source of the floating point values (sometimes) in this index.
(Per main comment thread)